packages/dx-react-chart/demos/area/streamgraph.md

```diff
@@ -0,0 +1,5 @@
+# React Chart Streamgraph
+
+The demo shows how to customize a series's offset and order using the [Stack](../../docs/reference/stack.md) plugin to create a Streamgraph chart.
```

Comment:
The demo shows how to use the [Stack](../../docs/reference/stack.md) plugin to customize a series's offset and order to create a Streamgraph chart.